bin/i18n/resources/dashboard/marketing_announcements/sync_in.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../../../dashboard/config/environment'
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_in_base'
+require_relative '../../../redact_restore_utils'
+require_relative '../../dashboard'
+require_relative '../marketing_announcements'
+
+module I18n
+  module Resources
+    module Dashboard
+      module MarketingAnnouncements
+        class SyncIn < I18n::Utils::SyncInBase
+          def process
+            progress_bar.progress = 50
+            prepare_marketing_announcements_data
+          end
+
+          private
+
+          def prepare_marketing_announcements_data
+            marketing_announcement_strings = {'banners' => {}}
+
+            banners = I18nScriptUtils.parse_file(ORIGIN_I18N_FILE_PATH)['banners']
+            banners.each do |banner_id, content|
+              marketing_announcement_strings['banners'][banner_id] = {
+                'title' => content['title'],
+                'body' => content['body'],
+                'buttonText' => content['buttonText']
+              }
+            end
+            I18nScriptUtils.write_json_file(I18N_SOURCE_FILE_PATH, marketing_announcement_strings)
```

Comment:
Shouldn't we convert the i18n data to the dashboard format before sending it to Crowdin as we do with the rest of the Dashboard i18n source files?
```suggestion
            I18nScriptUtils.write_json_file(I18N_SOURCE_FILE_PATH, I18nScriptUtils.to_dashboard_i18n_data('en', 'marketing_announcements', marketing_announcement_strings))
```

Reply:
I am not sure about this. I have seen in that yml files because we copy them from the dashboard/config.
and then we need to update the locale code for translations, which will require to dig the data content of the json, and still using in the sync out the `I18nScriptUtils.to_dashboard_i18n_data` method.